test/GoodGhosting_Send_AToken_Externally.test.js

```diff
@@ -24,7 +24,6 @@ contract("GoodGhosting_Send_AToken_Externally", (accounts) => {
     if (
         ![
             "local-mainnet-fork",
-            "local-celo-fork",
```

Comment:
just wondering, why are we removing celo network form these forked tests?

Reply:
I'll add might've removed by mistake